src/sdk/bucketspace_fs.js

```diff
@@ -202,55 +202,28 @@ class BucketSpaceFS extends BucketSpaceSimpleFS {
             }
             throw this._translate_bucket_error_codes(err);
```

Comment:
Would you please move lines 195 to 203 to a function?
The whole logic is just for getting the entries, not the "core" of the list buckets operation.

Reply:
this needs to be done as part of a refactoring PR, I see we can have this function used by health.js and manage_nsfs.js